katello/rubygem-foreman_pipeline/rubygem-foreman_pipeline.spec

```diff
@@ -0,0 +1,114 @@
+# This package contains macros that provide functionality relating to
```

Comment:
How well maintained is this plugin? Last update to the code was more than a year ago.

Reply:
I don't know as the Katello team was not maintaining it but since it requires Katello it had to live in our repositories. Do we have a policy for dropping plugins?

Reply:
Not sure about a policy. @xprazak2 could you comment on the status of this plugin?

Reply:
It is unmaintained since 1.13. I have no objections against droping it.